tests/unit/python/execution_tree/eval.py

```diff
@@ -40,10 +40,19 @@ def fib(n):
     else:
         return fib(n-1)+fib(n-2)
 
+assert fib.__physl_src__ == \
+    'block#1#0(define#1#0(fib#1#0, n#1#8, ' + \
+        'if(n#2#7 < 2, n#3#15, ' + \
+            '(fib((n#5#19 - 1)) + fib((n#5#28 - 2))))' + \
+        '), fib#1#0)\n'
+
 assert fib(10)[0] == 55.0
 
 @phyfun
 def pass_str(a):
     return a
 
+assert pass_str.__physl_src__ == \
+    'block#1#0(define#1#0(pass_str#1#0, a#1#13, a#2#11), pass_str#1#0)\n'
+
 assert "foo" == str(pass_str("foo"))
```

Comment:
Likewise, you probably want to copy the assertion to run before as well as after the src change.